test/test_clicommand.py

```diff
@@ -132,3 +133,23 @@ def test_bad_range_options(self):
                            "profilename", "hosts",
                            "a:d:b:s", "--auths",
                            "auth_1", "auth2"])
+
+    def test_a_auth_add(self):
```

Comment:
I'll open an issue regarding this. Currently the existing test cases depended on the fact that the files `data/credentials` and `data/profiles` are checked into git, which they shouldn't be.